### ISSUE TITLE
Q3D not closing due to GRPC plugin error

### ISSUE THREAD

**Issue by ToInoJo** [ansys/pyaedt#4945]
### Before submitting the issue

- [X] I have searched among the existing issues
- [X] I am using a Python virtual environment

### Description of the bug

When using PyAEDT 0.99 and Electronic Desktop 2022 R2, the opened desktop session does not close.
Instead, the following error appears (first File name changed to "..." for privacy):

> PyAEDT ERROR: **************************************************************
> PyAEDT ERROR:   File "...", line 1440, in <module>
> PyAEDT ERROR:     q.close_desktop()
> PyAEDT ERROR:   File "C:\Users\USR\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\pyaedt\application\Design.py", line 2488, in close_desktop
> PyAEDT ERROR:     self.release_desktop()
> PyAEDT ERROR:   File "C:\Users\USR\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\pyaedt\application\Design.py", line 2542, in release_desktop
> PyAEDT ERROR:     self.desktop_class.release_desktop(close_projects, close_desktop)
> PyAEDT ERROR:   File "C:\Users\USR\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\pyaedt\desktop.py", line 1583, in release_desktop
> PyAEDT ERROR:     self.grpc_plugin.recreate_application(True)
> PyAEDT ERROR: 'desktop' object has no attribute 'grpc_plugin' on release_desktop
> PyAEDT ERROR: Method arguments: 
> PyAEDT ERROR:     close_projects = True 
> PyAEDT ERROR:     close_on_exit = True 
> PyAEDT ERROR: **************************************************************

It does look like a lack of compatibility between PyAEDT 0.99 and EDB 2022 R2, but as far as I can see the 2022 R2 should be supported.

### Steps To Reproduce

1. Start the desktop session with Q3D:
`q = Q3d(specified_version="2022.2", non_graphical=NonGraphical, new_desktop_session=True, projectname='WFC_Check3', close_on_exit=False)`
2. Try to close the session with:
`q.close_desktop()`
3. Observe an error:
'desktop' object has no attribute 'grpc_plugin' on release_desktop

### Which Operating System are you using?

Windows

### Which Python version are you using?

3.10

### Installed packages

absl-py==1.4.0
aiohttp==3.8.4
aiosignal==1.3.1
altair==5.1.2
annotated-types==0.5.0
ansys-pythonnet==3.1.0rc3
anyio==4.0.0
appdirs==1.4.4
argon2-cffi==23.1.0
argon2-cffi-bindings==21.2.0
arrow==1.2.3
asttokens==2.3.0
astunparse==1.6.3
async-lru==2.0.4
async-timeout==4.0.2
attrs==23.1.0
Babel==2.12.1
backcall==0.2.0
beautifulsoup4==4.12.2
bleach==6.0.0
blinker==1.6.3
blis==0.7.11
cachetools==5.3.0
catalogue==2.0.10
certifi==2023.5.7
cffi==1.15.1
charset-normalizer==3.1.0
click==8.1.7
clipboard==0.0.4
clr-loader==0.2.6
colorama==0.4.6
comm==0.1.4
confection==0.1.3
contourpy==1.0.7
cycler==0.11.0
cymem==2.0.8
datasets==2.14.6
dearpygui==1.9.1
debugpy==1.6.7.post1
decorator==5.1.1
defusedxml==0.7.1
dill==0.3.7
docker-pycreds==0.4.0
docutils==0.20
en-core-web-md @ https://github.com/explosion/spacy-models/releases/download/en_core_web_md-3.6.0/en_core_web_md-3.6.0-py3-none-any.whl#sha256=1618fc62e7aedc2b778caa3ea6e9c8280beac3b83fe7b93316cbb9e8f3abe7c5
en-core-web-sm @ https://github.com/explosion/spacy-models/releases/download/en_core_web_sm-3.6.0/en_core_web_sm-3.6.0-py3-none-any.whl#sha256=83276fc78a70045627144786b52e1f2728ad5e29e5e43916ec37ea9c26a11212
exceptiongroup==1.1.3
executing==1.2.0
fastjsonschema==2.18.0
fCWT==0.1.18
filelock==3.12.2
flatbuffers==23.3.3
fonttools==4.39.3
fpdf2==2.7.9
fqdn==1.5.1
frozenlist==1.3.3
fsspec==2023.6.0
gast==0.4.0
gitdb==4.0.11
GitPython==3.1.40
google-auth==2.17.3
google-auth-oauthlib==1.0.0
google-pasta==0.2.0
grpcio==1.54.0
h5py==3.8.0
hmmlearn==0.3.0
huggingface-hub==0.17.3
idna==3.4
imageio==2.31.1
importlib-metadata==6.8.0
ipykernel==6.25.1
ipython==8.15.0
isoduration==20.11.0
jax==0.4.8
jedi==0.19.0
Jinja2==3.1.2
joblib==1.3.2
json5==0.9.14
jsonpointer==2.4
jsonschema==4.19.0
jsonschema-specifications==2023.7.1
jupyter-events==0.7.0
jupyter-lsp==2.2.0
jupyter_client==8.3.1
jupyter_core==5.3.1
jupyter_server==2.7.3
jupyter_server_terminals==0.4.4
jupyterlab==4.0.5
jupyterlab-pygments==0.2.2
jupyterlab_server==2.24.0
keras==2.12.0
keyboard==0.13.5
Kivy==2.1.0
kivy-deps.angle==0.3.3
kivy-deps.glew==0.3.1
kivy-deps.sdl2==0.4.5
Kivy-Garden==0.1.5
kiwisolver==1.4.4
langcodes==3.3.0
lazy_loader==0.2
libclang==16.0.0
lightning-utilities==0.9.0
Markdown==3.4.3
markdown-it-py==3.0.0
MarkupSafe==2.1.2
matplotlib==3.7.1
matplotlib-inline==0.1.6
mdurl==0.1.2
mistune==3.0.1
ml-dtypes==0.1.0
mpmath==1.3.0
multidict==6.0.4
multiprocess==0.70.15
munkres==1.1.4
murmurhash==1.0.10
mypy==1.2.0
mypy-extensions==1.0.0
nbclient==0.8.0
nbconvert==7.8.0
nbformat==5.9.2
nest-asyncio==1.5.7
networkx==3.1
nltk==3.8.1
notebook==7.0.3
notebook_shim==0.2.3
numpy==1.23.5
oauthlib==3.2.2
opencv-python==4.8.0.76
opt-einsum==3.3.0
overrides==7.4.0
packaging==23.1
pandas==2.1.0
pandocfilters==1.5.0
parso==0.8.3
pathtools==0.1.2
pathy==0.10.2
pickleshare==0.7.5
Pillow==9.5.0
platformdirs==3.5.1
plumbum==1.8.1
ply==3.11
pooch==1.7.0
portalocker==2.8.2
preshed==3.0.9
prometheus-client==0.17.1
prompt-toolkit==3.0.39
protobuf==4.23.0
psutil==5.9.5
pure-eval==0.2.2
pyaedt==0.9.9
pyarrow==13.0.0
pyasn1==0.5.0
pyasn1-modules==0.3.0
pycparser==2.21
pydantic==2.8.2
pydantic_core==2.20.1
pydeck==0.8.1b0
pyedb==0.20.0
Pygments==2.15.1
PyLTSpice==4.1.1
pyparsing==3.0.9
pyperclip==1.8.2
pypiwin32==223
PySimpleGUI==4.60.5
PySpice==1.5
python-dateutil==2.8.2
python-json-logger==2.0.7
pythonnet==3.0.1
pytomlpp==1.0.13
pytorch-lightning==2.0.4
pytz==2023.3.post1
pyvista==0.39.1
PyWavelets==1.4.1
pywin32==306
pywinpty==2.0.11
PyYAML==6.0
pyzmq==25.1.1
referencing==0.30.2
regex==2023.8.8
requests==2.30.0
requests-oauthlib==1.3.1
rfc3339-validator==0.1.4
rfc3986-validator==0.1.1
rich==13.6.0
rpds-py==0.10.0
rpyc==6.0.0
rsa==4.9
Rtree==1.3.0
safetensors==0.3.3
scikit-image==0.21.0
scikit-learn==1.3.1
scikit-rf==0.30.0
scipy==1.10.1
scooby==0.7.2
Send2Trash==1.8.2
sentence-transformers==2.2.2
sentencepiece==0.1.99
sentry-sdk==1.32.0
seqeval==1.2.2
setproctitle==1.3.3
simpletransformers==0.64.3
six==1.16.0
skweak==0.3.3
smart-open==6.4.0
smmap==5.0.1
sniffio==1.3.0
snorkel==0.9.9
soupsieve==2.5
spacy==3.6.1
spacy-legacy==3.0.12
spacy-loggers==1.0.5
srsly==2.4.8
stack-data==0.6.2
streamlit==1.28.0
sympy==1.12
tenacity==8.2.3
tensorboard==2.12.3
tensorboard-data-server==0.7.0
tensorflow==2.12.0
tensorflow-estimator==2.12.0
tensorflow-intel==2.12.0
tensorflow-io-gcs-filesystem==0.31.0
termcolor==2.3.0
terminado==0.17.1
textblob==0.17.1
thinc==8.1.12
threadpoolctl==3.2.0
tifffile==2023.4.12
tinycss2==1.2.1
tokenizers==0.13.3
toml==0.10.2
tomli==2.0.1
toolz==0.12.0
torch==2.0.1
torch-tb-profiler==0.4.3
torchmetrics==0.11.4
torchvision==0.15.2
tornado==6.3.3
tqdm==4.65.0
traitlets==5.9.0
transformers==4.33.3
typer==0.9.0
typing_extensions==4.8.0
tzdata==2023.3
tzlocal==5.2
uri-template==1.3.0
urllib3==2.0.2
validators==0.22.0
vtk==9.2.6
wandb==0.15.12
wasabi==1.1.2
watchdog==3.0.0
wcwidth==0.2.6
webcolors==1.13
webencodings==0.5.1
websocket-client==1.6.2
Werkzeug==2.3.4
wrapt==1.14.1
wrench==0.9.1
xxhash==3.4.1
yarl==1.9.2
zipp==3.17.0

**Comment by maxcapodi78** [comment]
This bug has been fixed. 
close_desktop is replaced by release_desktop
